src/debugger/Stepper.js

```diff
@@ -57,3 +58,42 @@ export class StepIntoStepper extends Stepper {
     return true;
   }
 }
+
+export class StepOverStepper extends Stepper {
+  constructor(filePath: string, line: number, column: number, stackSize: number) {
+    super(filePath, line, column);
+    this._startStackSize = stackSize;
+  }
+  _startStackSize: number;
+
+  isComplete(ast: BabelNode, realm: Realm): boolean {
+    // we should only step to statements
+    if (!IsStatement(ast)) return false;
+    let loc = ast.loc;
+    if (!loc) return false;
+    let filePath = loc.source;
+    let line = loc.start.line;
+    let column = loc.start.column;
+    if (!filePath) return false;
+    if (this._stepStartData) {
+      if (
+        filePath === this._stepStartData.filePath &&
+        line === this._stepStartData.line &&
+        column === this._stepStartData.column
+      ) {
+        return false;
+      }
+    } else {
+      return false;
+    }
+    if (realm.contextStack.length <= this._startStackSize) {
```

Comment:
It's bad to pass in the whole realm just for a single stack depth check. Just pass in realm.contextStack.length as "currenStackDepth"